packages/editor/src/editor/browser/cms/AddCmsQuickAction.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+import { useMeta } from '../../../context/useMeta';
+import { Button, Flex, Popover, PopoverArrow, PopoverContent, PopoverTrigger, toast } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useAppContext } from '../../../context/AppContext';
+import { useFunction } from '../../../context/useFunction';
+import { useQueryClient } from '@tanstack/react-query';
+import { genQueryKey } from '../../../query/query-client';
+
+export const AddCmsQuickActionPopover = ({
+  value,
+  onChange,
+  savedSelection,
+  inputRef
+}: {
+  value: string;
+  onChange: (value: string) => void;
+  savedSelection: { start: number; end: number };
+  inputRef: React.RefObject<HTMLInputElement>;
+}) => {
+  const [open, setOpen] = useState(false);
+
+  const { context } = useAppContext();
+  const queryClient = useQueryClient();
+  const cmsQuickActions = useMeta('meta/cms/cmsQuickActions', { context: { context, requiredProjects: false }, text: value }, []).data;
```

Comment:
requiredProjects needed for this request?

Reply:
In the core I reuse as context the CmsMetaRequest (which is needed for the cmsTree) which contains both FormContext and this requiredProjects flags. But actually I don't need the flag, because I don't do anything with it in the core. I change this so that only the formcontext has to be sent.